### ISSUE TITLE
Cannot set size of Icon or clearIcon in SearchBar

### ISSUE THREAD

**Issue by AkshitSahani** [react-native-training/react-native-elements#1047]
### Description

1. Explain what you did

I tried to set a larger size to the icon or clearIcon within my search bar. I see in the docs that you can provide a style key in both the icon and clearIcon objects. I tried setting different heights and widths 
 as an object for the style key.

2. What you expected to happen

I expected the width and size properties to change size of the icon (or clearIcon).

3. What actually happens

The provided width and height are applied to the container of the icon, not the icon itself. Any properties within the style object for icon or clearIcon are applied to the container and NOT the icon itself 

`react-native-elements` version: 5.6.0.

### Reproduction Steps and Sample Code

https://snack.expo.io/r1qtujqcG

---------------------

```
import React from 'react';
import {View, Text, } from 'react-native';
import {FormInput, Button, Icon, SearchBar} from 'react-native-elements';

MainHeader = () => {
  return(
    <View style={{width: '100%',  backgroundColor: '#426E86', height: '16%', flexDirection: 'column', justifyContent: 'center'}}>
      <View style={{flexDirection: 'row', alignItems: 'center', marginTop:'2%', justifyContent: 'space-between', width: '90%', marginLeft: '3%'}}>
        <Icon
          type="simple-line-icon"
          name="menu"
          color="#F9BA32"
          size={32}
        />

        <SearchBar
          placeholder="Capability"
          icon={{ type: 'evilicon', name: 'search', size:'32' }}
          inputStyle={{backgroundColor: 'white', fontSize: 18, height: 40, color: 'black'}}
          containerStyle={{backgroundColor: '#426E86', borderWidth: 0, borderTopWidth: 0, borderBottomWidth: 0, width: '75%', marginLeft: 10, marginRight: -5}}
          clearIcon={{name: 'close-o', style:{borderWidth: 1, borderColor: 'red', height: 30, width: 30,}}}
        />

        <Button
          title="GO"
          buttonStyle={{borderRadius: 5, backgroundColor: '#F9BA32', marginHorizontal: 0}}
          textStyle={{fontWeight:'bold', marginHorizontal:0}}
        />
      </View>

      <View style={{flexDirection: 'row', justifyContent: 'space-between', alignItems: 'center', width: '80%', marginLeft: '16%'}}>
        <Text style={{color: '#F9BA32', fontSize: 18, fontWeight: 'bold'}}>Electrician</Text>
        <Text style={{color: '#F9BA32', fontSize: 18, fontWeight: 'bold'}}>Carpenter</Text>
        <Text style={{color: '#F9BA32', fontSize: 18, fontWeight: 'bold'}}>Plumber</Text>
      </View>
    </View>
  )
};

export default MainHeader;
```
-------------------
See the code for the SearchBar component within the MainHeader. I see both the icons (icon and clearIcon) correctly. However, I want my searchbar to be bigger so I have given it a large height. However the icons look too small now with this new searchbar height, so I wanted to increase size of the icons as well. I tried various things, but none have been able to get the desired effect of increasing size of the icons.

Thanks

**Comment by xavier-villelegier** [comment]
Hey @AkshitSahani, this is not possible for now ! If anyone wants to tackle this issue, the hard coded size is [here](https://github.com/react-native-training/react-native-elements/blob/master/src/searchbar/SearchBar-default.js#L105) !

**Comment by Gregjarvez** [comment]
@xavier-villelegier requires that I just do this right 

```javascript 
          <Icon
            size={icon.size || 16}
            style={[styles.icon, styles.searchIcon, icon.style && icon.style]}
            name={icon.name || 'search'}
            color={icon.color || colors.grey3}
          />
```

**Comment by xavier-villelegier** [comment]
Yes exactly

**Comment by iRoachie** [comment]
Merged in https://github.com/react-native-training/react-native-elements/commit/84ffc9cdeae21b261d786c1e413b084c2a12fa91 will be available next release

**Comment by AkshitSahani** [comment]
Thanks guys! Appreciate the help :)

**Comment by iRoachie** [comment]
Just released [`1.0.0-beta4`](https://github.com/react-native-training/react-native-elements/releases/tag/v1.0.0-beta4). You can now set the size of the icons!
